.circleci/config.yml

```diff
@@ -75,23 +79,28 @@ jobs:
       - run: make install-tools
       - run: make lint
   test:
+    resource_class: large
     machine:
       image: ubuntu-2004:202107-02
     steps:
       - checkout
+      - go/install:
+          version: "1.16.2"
```

Comment:
Shouldn't it be 1.17.x ?

Reply:
We do not have switched to go 1.17 builds so far for driftct, why do we want to run our test not to the target build version not sure to get it ? 🤔

Reply:
I forgot we didn't move to 1.17 yet, nvm